samples/iframe/package.json

```diff
@@ -4,14 +4,15 @@
     "private": true,
     "scripts": {
         "build": "vertigis-web-sdk build",
-        "start": "vertigis-web-sdk start",
+        "start": "cross-env OPEN_PAGE=/parent.html vertigis-web-sdk start",
```

Comment:
If we have this, do we still need the `?initialpath=/parent.html` logic in `App.tsx` ?

Reply:
Unfortunately, no. Codesandbox doesn't support opening the browser from the tooling. They open to root always it appears unless specified using the `initialPath` query param. This highlights the fact that when forking the iframe sample, you're kinda stuck unless you know to add the param.

Reply:
See https://github.com/codesandbox/codesandbox-client/issues/1442

Reply:
Ahh too bad.